arrow-udf-wasm/CHANGELOG.md

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [0.4.0] - 2024-10-10
+## [0.4.0] - 2024-11-15
```

Comment:
Why the release date is changed? Did `0.4.0` "republished"?

Reply:
The change of this PR first appeared in 0.4.1. Seems that this line should be reverted.